cirq-core/cirq/circuits/_block_diagram_drawer.py

```diff
@@ -158,9 +158,9 @@ class BlockDiagramDrawer:
     """Aligns text and curve data placed onto an abstract 2d grid of blocks."""
 
     def __init__(self):
-        self._blocks = collections.defaultdict(Block)  # type: Dict[Tuple[int, int], Block]
-        self._min_widths = collections.defaultdict(lambda: 0)  # type: Dict[int, int]
-        self._min_heights = collections.defaultdict(lambda: 0)  # type: Dict[int, int]
+        self._blocks = collections.defaultdict(Block)  # type: dict[tuple[int, int], Block]
```

Comment:
why the change here? I think this is only valid for newer pythons. We still support 3.7 (and maybe 3.6??)

Reply:
Yes, you are right.

I had just changed `Dict` to `dict` keeping in mind the current Python guidelines (Python 3.9 calls it `dict` and `tuple`).
I have reverted them for the time being.